setup/autoinstall-iso/iso-create.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+set -x
+
+#################################################################
+# This file creates a custom ISO for the AI Pong Exhibit based
+# off of the open source Ubuntu Server ISO. For more information
+# look at the ISO Creation Documentation under the docs directory
+# where this file is located. (/autoinstall-iso/docs)
+# Modified: 3/12/2025
+#################################################################
+
+############
+# Variables
+############
+ISO_DIRECTORY='iso-test'
+
+# Prerequisites
+# p7zip - for unpacking the source ISO
+# wget - to download fresh build of Ubuntu Server ISO from online
+# xorriso - for building the modified ISO
+sudo apt update
+sudo apt install p7zip wget xorriso
```

Comment:
```suggestion
sudo apt install p7zip-full wget xorriso --install-suggests
```
I had to do this to get 7z working, only had 7za with this command as it was. However, I'm on a funky LXC version of deb so ignore if you are confident